assets/app/views/browse/build.html

```diff
@@ -6,84 +6,26 @@
       <div class="row">
         <div class="col-md-12">
           <div class="tile">
-            <dl class="dl-horizontal left">
-              <dt>Name:</dt>
-              <dd>{{build.metadata.name}}</dd>
-              <dt>Status:</dt>
-              <dd>
-                <span ng-switch="build.status.phase" class="hide-ng-leave">
-                  <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-                  <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true"></span>
-                </span>
-                {{build.status.phase}}
+            <h1>Build {{build.metadata.name}}</h1>
+            <tabset>
+              <tab active="selectedTab.details">
+                <tab-heading>Details</tab-heading>
+                <ng-include src=" '/views/browse/_build-details.html' "></ng-include>
```

Comment:
@benjaminapetersen Please make sure you pick up any changes from @jwforres to build.html and pod.html before this PR is merged since you've moved the details into separate files.